src/ocm/components/clusterConfiguration/networkConfiguration/NetworkConfigurationForm.tsx

```diff
@@ -58,19 +59,20 @@ const NetworkConfigurationForm: React.FC<{
     useFormikContext<NetworkConfigurationValues>();
   const isAutoSaveRunning = useFormikAutoSave();
   const errorFields = getFormikErrorFields(errors, touched);
+  const { supportedPlatformIntegration } = useClusterSupportedPlatforms(cluster.id);
```

Comment:
Burn the whole cluster in deepest levels of hell. /s

Anyway, I think this is something we should discuss on a broader level. Perhaps we shouldn't allow users past the Host discovery step if their hosts are of different types or something... Currently, there is nothing we can do in the UI if that happens - we do not have enough information.

Reply:
Based on the conversion in Slack, looks like the code would work well, as we would never receive the value if not all hosts are of the same platform. However, I think we can implement it in a way that it would still work if the BE changes its behaviour.

So I would avoid using `supportedPlatformIntegration` which is guaranteed to be exclusive, and instead to look at all of the `supported-platforms`, and if `nutanix` is present, perform the logic.

Up to you if you feel it's needed or not.